File: app/src/main/java/com/riders/thelab/data/remote/dto/directions/Directions.java
package com.riders.thelab.data.remote.dto.directions;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;


public class Directions {

    @Expose
    ArrayList<Route> route;

    @Expose
    String status;

    @SerializedName("overview_polyline")
    String overviewPolyline;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/directions/Directions.kt
package com.riders.thelab.data.remote.dto.directions

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Directions(
    @Expose
    var route: ArrayList<Route>? = null,

    @Expose
    var status: String? = null,

    @SerializedName("overview_polyline")
    var overviewPolyline: String? = null
)